dbscripts/xml/upgrade.xml

```diff
@@ -11,7 +11,7 @@
   *
   -->
 
-<install version="2.4.6.0">
+<install version="2.4.6.1">
```

Comment:
The convention that we're following for releases is this:
- Builds (2.4.6-0 to 2.4.6-1) are released periodically from the stable git branch (ojs-stable-2_4_6) so that users who aren't hosting with the git branch directly can receive periodic updates.
- Revisions (2.4.6 to 2.4.7) involve creating a new branch (ojs-stable-2_4_7), from which new builds will be periodically released.

Because of that, we typically wouldn't permit any database structure changes in builds. Those would need a new revision. So I'd suggest this be released in OJS 2.4.7 rather than 2.4.6-1.